models/collectionPoint.js

```diff
@@ -1,48 +1,42 @@
-"use strict";
-
-const Event = require("./event");
-const User = require("./user");
+'use strict';
 
 module.exports = (sequelize, DataTypes) => {
   const collectionPoint = sequelize.define(
-    "collectionPoint",
+    'collectionPoint',
     {
       name: DataTypes.STRING,
       eventId: DataTypes.INTEGER,
-      //TODO: Add Location (coordinate)
+      // TODO: Add Location (coordinates)
     },
     {
       paranoid: true,
       hooks: {
-        afterDestroy: function (instance, options) {
-          instance.getPatients().then((patients) =>
-            patients.map((patient) => {
-              patient.destroy();
-            })
-          );
+        afterDestroy: (instance) => {
+          instance
+            .getPatients()
+            .then((patients) => patients.map((patient) => patient.destroy()));
         },
-        afterRestore: function (instance, options) {
+        afterRestore: (instance) => {
           instance
             .getPatients({ paranoid: false })
             .then((patients) => patients.map((patient) => patient.restore()));
         },
       },
     }
   );
-  collectionPoint.associate = function (models) {
-    // associations can be defined here
+  collectionPoint.associate = (models) => {
     collectionPoint.hasMany(models.patient, {
       hooks: true,
     });
 
-    collectionPoint.belongsTo(User(sequelize, DataTypes), {
-      foreignKey: "createdBy",
-      targetKey: "id",
+    collectionPoint.belongsTo(models.user, {
```

Comment:
refactored so that the function arg `models` was being used and we wouldn't need to import `User` separately